client/my-sites/plans-v2/plans-filter-bar/index.tsx

```diff
@@ -34,12 +44,48 @@ interface Props {
 	duration?: Duration;
 	productType?: ProductType;
 	onDurationChange?: ( arg0: Duration ) => void;
-	onProductTypeChange?: ( arg0: Duration ) => void;
+	onProductTypeChange?: ( arg0: ProductType ) => void;
 }
 
 const CALYPSO_MASTERBAR_HEIGHT = 47;
 const CLOUD_MASTERBAR_HEIGHT = 94;
 
+const REALTIME_PRODUCTS = [ PLAN_JETPACK_SECURITY_REALTIME, PRODUCT_JETPACK_BACKUP_REALTIME ];
+
+const DiscountMessage = () => {
+	const translate = useTranslate();
+
+	const croVersion = getJetpackCROActiveVersion();
+	const slugsToCheck = [
```

Comment:
I'm not super happy with the way I'm bringing in products and plans here, because it's not strongly tied to what we're showing on the product grid; but short of some major re-design to both this component and `ProductGridAlt`/`SelectorPageAlt`, I'm not sure what better options exist. 🙁